Source/Krypton Components/Krypton.Toolkit/Designers/Action Lists/KryptonTextBoxActionList.cs

```diff
@@ -58,13 +58,13 @@ public KryptonTextBoxActionList(KryptonTextBoxDesigner owner)
         /// </summary>
         public PaletteMode PaletteMode
         {
-            get => _textBox.PaletteMode;
+            get => _textBox!.PaletteMode;
 
             set
             {
-                if (_textBox.PaletteMode != value)
+                if (_textBox?.PaletteMode != value)
```

Comment:
if textbox is null and value is not, then line 68 will fail

Reply:
Reverted